engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js

```diff
@@ -122,7 +122,13 @@ angular.module('Bastion.content-views').config(['$stateProvider', function ($sta
         controller: 'ContentViewVersionContentController',
         templateUrl: 'content-views/versions/views/content-view-version-puppet-modules.html'
     })
-
```

Comment:
This blank line seems to be a logical separation in the states. Wondering if it should be kept but placed after your new state.